model/riscv_pmp_regs.sail

```diff
@@ -187,9 +189,10 @@ function pmpWriteCfgReg(n, v) = {
                 pmp13cfg = pmpWriteCfg(pmp13cfg, v[15..8]);
                 pmp14cfg = pmpWriteCfg(pmp14cfg, v[23..16]);
                 pmp15cfg = pmpWriteCfg(pmp15cfg, v[31..24]);
-              }
+              },
+         _ => internal_error("Unexpected pmp config reg write")
        }
-  else if sizeof(xlen) == 64
+  else if sizeof(xlen) >= 64
```

Comment:
I cannot see how this matter in the current model since XLEN is never configured >64 and changing it here might give confidance that its correctly implemented else where ?

Reply:
It doesn't matter at the moment but RV128 is in the spec. and at some point the model might have to support it.